frontend/src/app/organization/organization-details/organization-details.component.ts

```diff
@@ -7,30 +7,19 @@
  * @license MIT
  */
 
-import { Component, inject } from '@angular/core';
+import { Component } from '@angular/core';
 import { ActivatedRoute, ActivatedRouteSnapshot, ResolveFn, Route } from '@angular/router';
 import { MatIconRegistry } from '@angular/material/icon';
 import { DomSanitizer } from '@angular/platform-browser';
-import { Observable, map } from 'rxjs';
 import { MatSnackBar } from '@angular/material/snack-bar';
 import { profileResolver } from '/workspace/frontend/src/app/profile/profile.resolver';
-import { Organization, OrganizationService } from '../organization.service';
-import { Profile, ProfileService } from '/workspace/frontend/src/app/profile/profile.service';
+import { Organization } from '../organization.service';
+import { Profile } from '/workspace/frontend/src/app/profile/profile.service';

```

Comment:
Maybe a stupid question but is there a reason for us to be importing Profile/use profile on this page?